configure.ac

```diff
@@ -1749,9 +1724,7 @@ else
     fi
     AC_MSG_RESULT($use_natpmp_default)
     AC_DEFINE_UNQUOTED([USE_NATPMP], [$natpmp_setting], [NAT-PMP support not compiled if undefined, otherwise value (0 or 1) determines default state])
-    if test "$TARGET_OS" = "windows"; then
-      NATPMP_CPPFLAGS="-DSTATICLIB -DNATPMP_STATICLIB"
-    fi
+    AS_CASE([$host_os], [*mingw*], [NATPMP_CPPFLAGS="-DSTATICLIB -DNATPMP_STATICLIB"])
```

Comment:
What about non-mingw windows? Does this make it more difficult to switch compilers later? E.g. does the clang windows tuple have mingw in it?